Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38000&attempt_id=532d3e30-ae2c-4962-802d-55f282cb248f)

Attempted: dovecot, dovecot_pigeonhole

The following builds were skipped because they don't evaluate on x86_64-linux: dovecotWithPgSQL

<details><summary>Partial log (click to expand)</summary><p>

```
libtool: link: gcc -shared  -fPIC -DPIC  .libs/managesieve-settings.o    -g -O2 -fstack-protector-strong   -Wl,-soname -Wl,libmanagesieve_settings.so -o .libs/libmanagesieve_settings.so
libtool: link: ( cd ".libs" && rm -f "libmanagesieve_settings.la" && ln -s "../libmanagesieve_settings.la" "libmanagesieve_settings.la" )
make[3]: Leaving directory '/tmp/nix-build-dovecot-pigeonhole-0.5.0.1.drv-0/dovecot-2.3-pigeonhole-0.5.0.1/src/managesieve'
make[2]: *** [Makefile:428: all-recursive] Error 1
make[2]: Leaving directory '/tmp/nix-build-dovecot-pigeonhole-0.5.0.1.drv-0/dovecot-2.3-pigeonhole-0.5.0.1/src'
make[1]: *** [Makefile:743: all-recursive] Error 1
make[1]: Leaving directory '/tmp/nix-build-dovecot-pigeonhole-0.5.0.1.drv-0/dovecot-2.3-pigeonhole-0.5.0.1'
make: *** [Makefile:602: all] Error 2
builder for ‘/nix/store/k37wmh0s8mk7rxzh4is8p8nmi4xgrwz6-dovecot-pigeonhole-0.5.0.1.drv’ failed with exit code 2
error: build of ‘/nix/store/k37wmh0s8mk7rxzh4is8p8nmi4xgrwz6-dovecot-pigeonhole-0.5.0.1.drv’ failed
```
</p></details>

--- a/nixos/modules/services/mail/dovecot.nix
+++ b/nixos/modules/services/mail/dovecot.nix
@@ -30,6 +30,7 @@ let
 
     ''
       default_internal_user = ${cfg.user}
+      default_internal_group = ${cfg.group}
       ${optionalString (cfg.mailUser != null) "mail_uid = ${cfg.mailUser}"}
       ${optionalString (cfg.mailGroup != null) "mail_gid = ${cfg.mailGroup}"}
 

--- a/pkgs/servers/mail/dovecot/plugins/pigeonhole/default.nix
+++ b/pkgs/servers/mail/dovecot/plugins/pigeonhole/default.nix
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "dovecot-pigeonhole-${version}";
-  version = "0.5.0.1";
+  version = "0.5.1";
 
   src = fetchurl {
     url = "http://pigeonhole.dovecot.org/releases/2.3/dovecot-2.3-pigeonhole-${version}.tar.gz";
-    sha256 = "1lpsdqh9pwqx917z5v23bahhhbrcb3y5ps3l413sli8cn4a6sdan";
+    sha256 = "0ivmaxic6cygfphvlrvy0xgggydm7j7kjv1ssfqbr08q4rcsmc73";
   };
 
   buildInputs = [ dovecot openssl ];

